front_end/webserver.py

```diff
@@ -417,9 +423,12 @@ def post(self, course, assignment):
                 if content.has_duplicate_title(content.get_assignments(course), assignment_basics["id"], title):
                     result = "Error: An assignment with that title already exists."
                 else:
-                    content.specify_assignment_basics(assignment_basics, title, visible)
-                    content.specify_assignment_details(assignment_details, introduction, None, datetime.datetime.now())
-                    assignment = content.save_assignment(assignment_basics, assignment_details)
+                    if len(title) > 50:
+                        result = "Error: Title cannot exceed 50 characters."
```

Comment:
Please change this to "Error: The title cannot exceed..." That way it uses valid grammar and is consistent with the error message above.